README.md

```diff
@@ -30,16 +30,80 @@ Please consider donating to support me and help me put more time into my plugins
   "Drop on Death (true/false)": true,
   "Erase on Death (true/false)": false,
   "Use Blacklist (true/false)": false,
-  "Clear Backpacks on Map-Wipe (true/false)": true,
+  "Clear Backpacks on Map-Wipe (true/false)": false,
```

Comment:
Hmm, this seems odd. I see we are just updating the documentation here, but we should at some point probably change the default to `true` since not clearing on map wipe would be a loop hole considering the default is also to drop on death.

Reply:
I took entire file created by default for this section. That's why it was changed. I'm not sure why settings are like they are, I joined here later :)